### DISCUSSION TITLE
Package Installation Error - packages not found in changelist==0.5

### DISCUSSION THREAD

**Discussion by akshitasure12** [networkx/networkx#7705] [Q&A] [answered]
Hi

I am new to open source environment and want to contribute to networkx project.

When i am trying to install getting package not available in channels error

conda install -c conda-forge --file requirements/developer.txt

PackagesNotFoundError: The following packages are not available from current channels:

  - changelist==0.5
  
Thanks
Akshita

**Comment by Schefflera-Arboricola** [comment]
Hi @akshitasure12 , welcome to open-source, and it's great to hear that you're interested in contributing to NetworkX! 

Thank you for reporting this! I don't think [changelist](https://github.com/scientific-python/changelist) is available through conda-forge. So, you can instead install it by running `pip install changelist==0.5`. And please feel free to create a PR to update this in the [Contributor's guide](https://networkx.org/documentation/latest/developer/contribute.html).

Thank you :)

**Comment by akshitasure12** [comment]
Thank you for your prompt response, I will raise a PR for the same.

**Reply by akshitasure12** [reply]
Planning to break this down into 3 sub-parts for easier understanding when errors are encountered.
In my case, it took time to understand in which file the issue was.

Current : 
conda install -c conda-forge --file requirements/default.txt --file requirements/test.txt --file requirements/developer.txt

Proposed :
conda install -c conda-forge --file requirements/default.txt
conda install -c conda-forge --file requirements/test.txt
pip install changelist==0.5 

Also, encountered another minor documentation issue with respect to pre-commit. 
I noticed that the "pip install pre-commit" command was missing that I am planning to add because it gave me a command not found error otherwise.

Current :
pre-commit install

Proposed : 
pip install pre-commit
pre-commit install

Pls let me know should i proceed with raising PR as per contributor gudielines.

**Reply by Schefflera-Arboricola** [reply]
refer discussion under https://github.com/networkx/networkx/pull/7706

PS: `changelist` is only used at the time of release of the package, I think. So, as a contributor you won't need it. So you can remove it locally from your `requirements/developer.txt` file and set up the environment until https://github.com/conda-forge/staged-recipes/pull/28089 is merged. And thank you for reporting this and creating the PR!

**Comment by MridulS** [comment, marked as answer]
@akshitasure12 could you try running the original instructions again? `changelist` should be available now :)
